ui/components/data-sources/AddDataSourceDialog.js

```diff
@@ -13,7 +14,7 @@ const INITIAL_STATE = {
     },
     postgresOptions: {
         url: "",
-        port: 5432,
+        port: DEFAULT_POSTGRES_PORT,
```

Comment:
I see your point here but there's really no need for extracting _yet another_ constant for this value. `INITIAL_STATE` is already the extracted constant defining all default values.

Reply:
👍 Perfect!